apps/hash-frontend/src/pages/login.page.tsx

```diff
@@ -140,7 +126,7 @@ const LoginPage: NextPageWithLayout = () => {
             }
 
             updateActiveWorkspaceAccountId(authenticatedUser.accountId);
-
+            // console.log("redirect to /");
```

Comment:
```suggestion

```

Reply:
Oops, thanks for catching! Tracking this one down was a pain 😅